### ISSUE TITLE
env in webpack.config.js shows undefined

### ISSUE THREAD

**Issue by nooowei** [Festivus-Music/web-app#68]
Already ran `npm config get production` and `npm config set -g production false`, all the devDependencies are installed and found in node_modules folder, such that nodemon is working. Also able to connect to MongoDB with URI provided in .env file. Any idea why?

**Comment by andrewpat24** [comment]
Yeah, that's on my end too. Pretty much, when we throw this thing on heroku, AWS, linode, or whatever, that env parameter from module exports it's going to be equal to "production" and set the devtool to source-map ([which is prod-compatible](https://webpack.js.org/configuration/devtool/#devtool). 

Which you can see here: 

`devtool: isProduction ? "source-map" : "inline-source-map"`

Now that you mention it, it feels pretty sloppy having it be a straight up 'undefined'. I did try fixing it but didn't deem it totally necessary since it's behaving like we want it to anyways. 

I'll just take that console.log out for now and push the change over to dev.

**Comment by nooowei** [comment]
Ok, in that case it's probably not the issue causing the page to not render on my end, I successfully connected it to the MongoDB Atlas, as well as Spotify API, but only the nav bar was rendered on my page. any idea what could potentially be the reason?
![issue1](https://user-images.githubusercontent.com/40618171/72645831-875f0e80-3942-11ea-9f55-fd1a56be47d9.png)

**Comment by andrewpat24** [comment]
Haha well.. that's because I haven't made it yet 😅

Does logging in work for you?

**Comment by nooowei** [comment]
Haha I see, I should've taken a look into the react files before I opened this issue. I will take a better look later today and see if I can help take down some issues!

**Comment by andrewpat24** [comment]
Awesome man! If anything's overly vague feel free to ping me or throw a comment on the issue so I can add extra clarification!
